config/routes.rb

```diff
@@ -28,4 +30,15 @@
     end
   end
   
-end
\ No newline at end of file
+  namespace :api do
+    namespace :v1 do
+      resources :users, only: [:index, :show] do 
+        resources :posts, only: [:index, :show, :new, :create, :destroy] do 
+          resources :comments, only: [:new, :create, :destroy]
```

Comment:
- [ ] Based on the poject requirement and based on your code on line 37, this endpoint that returns **_all comments_** for a post is missing because you didn't specify the `:index,` 

 `GET     /api/v1/users/:user_id/posts/:post_id/comments      api/v1/comments#index`

To fix this, you add the `:index` to the comment routes on line 37. 👇🏾 

```ruby
Rails.application.routes.draw do
  namespace :api do
    namespace :v1 do
      resources :users, only: [:index, :show] do 
        resources :posts, only: [:index, :show, :new, :create, :destroy] do 
          resources :comments, only: [:index, :new, :create, :destroy]
          resources :likes, only: [:index, :create]
        end        
      end
    end
  end
end
```